.github/workflows/zimbabwe_staging.yml

```diff
@@ -0,0 +1,127 @@
+# This is a basic workflow to publish a Twilio function with Github Actions
+
+name: Childline Staging release
+
+# Controls when the action will run.
+on:
+  workflow_dispatch
+  
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      # Setup credentials to access AWS for parameters
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+      # Get AWS parameters to setup environment variables for Serveless function
+      - name: Set Twilio Sync API key
+        uses: "marvinpinto/action-inject-ssm-secrets@latest"
+        with:
+          ssm_parameter: "STG_TWILIO_Zw-v1_API_KEY"
```

Comment:
Same issue as in https://github.com/techmatters/flex-plugins/pull/849 with the naming being wrong in the workflow (probably this parameters don't even exist in Parameter Store, I recommend manually checking there). Same is true for all the parameters in this PR (probably you generated the worflows with the Zw-v1 name "short helpline" variable :P)

Reply:
@GPaoloni Sorry, that was a clumsy mistake. Fixed